openmp/libomptarget/DeviceRTL/src/State.cpp

```diff
@@ -202,6 +204,7 @@ void state::ICVStateTy::assertEqual(const ICVStateTy &Other) const {
 
 void state::TeamStateTy::init(bool IsSPMD) {
   ICVState.NThreadsVar = 0;
+  ICVState.NTeamsVar = 0;
```

Comment:
I don't see how this var is updated among different target regions, so is it necessary to make it part of the team state? Also, I thought this value should be related to the nested level.

Reply:
I don't think right now we can update it on the device. That said, once teams doesn't have to be nested strictly, one might be able to set it. I'm fine with an ICV for now. The initialization, via OMP_NUM_TEAMS_DEV_<N>, is going to be a follow up.

Reply:
If that is the case, do we need to put it into team state?